doc/modules/neural_networks_supervised.rst

```diff
@@ -270,7 +270,7 @@ function for classification is Cross-Entropy, which in binary case is given as,
 
 .. math::
 
-    Loss(\hat{y},y,W) = -y \ln {\hat{y}} - (1-y) \ln{(1-\hat{y})} + \alpha ||W||_2^2
+    Loss(\hat{y},y,W) = -\dfrac{1}{n}\sum_{i=0}^n(y_i \ln {\hat{y_i}} + (1-y_i) \ln{(1-\hat{y_i})}) + \dfrac{\alpha}{2n} ||W||_2^2
```

Comment:
> The original issue was opened because it was unclear that the regularization was also scaled

Ok then I guess it's good to be extra explicit.

> I think it's used there to cancel out the 2 in the gradient.

Yes it's pretty standard with the l2 regularization